modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/TokenCountFieldMapper.java

```diff
@@ -124,6 +125,11 @@ public ValueFetcher valueFetcher(SearchExecutionContext context, String format)
             }
             return new DocValueFetcher(docValueFormat(format, null), context.getForField(this, FielddataOperation.SEARCH));
         }
+
+        @Override
+        public BlockLoader blockLoader(BlockLoaderContext blContext) {
+            return null;
```

Comment:
`token_count` is not supported in ESQL but `TokenCountFieldType` inherits `blockLoader` implementation from `NumberFieldType` which leads to fun test failures. IMO it is more logical to do this until it is actually supported.

Reply:
Okay, that was not a good idea it seems.

Reply:
Tests are okay, i was just confused.